pipeline/data_merging/aggregate_across_columns.py

```diff
@@ -209,27 +213,32 @@ def selectMaxAlleleFrequency(newRow):
                 maxFreq = aa
                 maxFreqString = "%f (AA from ESP)" % aa
     if newRow["EUR_Allele_frequency_1000_Genomes"] != EMPTY:
-        eur_af = float(newRow["EUR_Allele_frequency_1000_Genomes"])
+        freqs = [float(i) for i in newRow["EUR_Allele_frequency_1000_Genomes"].split(',')]
```

Comment:
This allows merging variants with different allele frequencies. Since `newRow["EUR_Allele_frequency_1000_Genomes"]` would be a list in this case, it cannot be converted to a float and compared without treating each item individually.